extensions/exchanges/gdax/exchange.go

```diff
@@ -42,6 +44,7 @@ func NewGDAXExchange(key, passphrase, secret string, httpClient *http.Client) *g
 	}
 }
 
+// GetTrades  returns coinbase(gdax) transaction history
```

Comment:
"returns the transaction history for a specific product on the Coinbase (gdax) exchange"